src/bot/core/music.py

```diff
@@ -0,0 +1,292 @@
+                await ctx.send(embed=MsgFormatter.get(ctx, 'Player Resumed', 'Now playing: ' + song_list[self.songQueue].title))
+            elif len(song_list) == self.songQueue:
+                await ctx.send(embed=MsgFormatter.get(ctx, 'No Song in Queue', 'The player doesn\'t have any song to play. Use //search to add songs in queue'))
+            else:
+                await self.playMusic(ctx)
+        else:
+            if isinstance(song, list):
+                song = " ".join(song)
+
+            if "youtube.com" in song:
+                r = requests.get(song)
+                soup = BeautifulSoup(r.text, 'lxml')
+                title = str(soup.find('title'))[7:-8]
+                song_list.append(Song.addSong(title, song))
+                await ctx.message.delete()
+                await ctx.send(embed=MsgFormatter.get(ctx, song_list[-1].title, " in Queue"))
+                await self.player(ctx)
+
+            else:
+                r = requests.get(
+                    "https://www.youtube.com/results?search_query=" + song)
+                soup = BeautifulSoup(r.text, 'lxml')
+                J = str(soup.find_all('script')[27])
+                J = J.split('var ytInitialData = ')[1].split(';<')[0]
+
+                s = json.loads(J)
+
+                a, b = 0, 0
+                while True:
+                    content = s['contents']['twoColumnSearchResultsRenderer']['primaryContents'][
+                        'sectionListRenderer']['contents'][0]['itemSectionRenderer']['contents']
+                    if ('videoRenderer' in content[a]) and (not 'badges' in content[a]['videoRenderer']):
+                        song_ = Song()
+                        song_.searchSong(content, a)
+                        song_list.append(song_)
+                        break
+                    a += 1
+                await ctx.message.delete()
+                await ctx.send(embed=MsgFormatter.get(ctx, song + ' searched', song_list[-1].title + '\n Length: ' + song_list[-1].length))
+                await self.player(ctx)
+
+    @commands.command(aliases=['pp'])
+    @MGCertificate.verify(level=Level.TRUSTED_USERS)
+    async def pause(self, ctx: commands.Context):
+        """
+        Pauses the song that is playing
+
+        //pause
+        //pp
+
+        Cannot pause the player if the player is already paused
+        """
+        if ctx.voice_client.is_playing():
+            ctx.voice_client.pause()
+            await ctx.send(embed=MsgFormatter.get(ctx, 'Player Paused', 'Paused at: ' + song_list[self.songQueue].title))
+        else:
+            await ctx.send(embed=MsgFormatter.get(ctx, 'Player Already Paused', 'Paused at: ' + song_list[self.songQueue].title))
+
+    @commands.command(aliases=['ss'])
+    @MGCertificate.verify(level=Level.TRUSTED_USERS)
+    async def skip(self, ctx: commands.Context):
+        """
+        Skips the song that is playing and plays the next song in queue
+
+        //skip
+        //ss
```

Comment:
```suggestion
        {commandPrefix}skip
        {commandPrefix}ss
```
Please use `{commandPrefix}` instead of arbitrary prefix in all help docs...